runelite-client/src/main/java/net/runelite/client/plugins/microbot/virewatch/PVirewatchKillerConfig.java

```diff
@@ -0,0 +1,269 @@
+package net.runelite.client.plugins.microbot.virewatch;
+
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
+import net.runelite.client.plugins.microbot.virewatch.models.PRAY_STYLE;
+
+@ConfigGroup("PVireKillerConfig")
+public interface PVirewatchKillerConfig extends Config {
+
+    @ConfigItem(
+            keyName = "guide",
+            name = "How to use",
+            description = "How to use this plugin",
+            position = 0
+    )
+    default String GUIDE() {
+        return "Read the description of the settings for more information. Start with inventory and gear setup near the prayer statue and configure the settings. \n HAVE AUTO RETALIATE ON!" ;
```

Comment:
We have a method in the Rs2Combat class, that enables auto retaliate, you can call this in your main loop to make sure that it's on

enableAutoRetialiate

Reply:
Done